app/api/helpers/feedback.py

```diff
@@ -0,0 +1,16 @@
+import logging
+from app.models import db
+
+
+def delete_feedback(feedback):
+    """
+    Delete the feedback if rating is zero
+    :param feedback: Feedback to be deleted
+    :return:
+    """
+    db.session.delete(feedback)
```

Comment:
Just a question, Where are we checking if the feedback is 0?

Reply:
@mrsaicharan1 when rating is set to 0 on FE, `destroyRecord()` is called which sets `deleted_at` to non null value and then the rest is handled by the checks I introduced -
```
if feedback and 'deleted_at' in data and data['deleted_at']:
            if has_access('is_user_itself', user_id=feedback.user_id):
                delete_feedback(feedback)
            else:
                raise ForbiddenException({'source': ''},
                                         "Feedback can be deleted only by user himself")
```